the-open-university-harvard.csl

```diff
@@ -151,16 +159,28 @@
       <key macro="author-short"/>
     </sort>
     <layout prefix="(" suffix=")" delimiter="; ">
-      <group delimiter=", ">
-        <group delimiter=", ">
-          <text macro="author-short"/>
-          <text macro="year-date"/>
-        </group>
-        <group>
-          <text term="page" form="short" suffix=" "/>
-          <text variable="locator"/>
-        </group>
-      </group>
+      <choose>
+        <!-- Specific formats (citation) -->
+        <if type="motion_picture">
+          <group delimiter=", ">
+            <text macro="title"/>
+            <text macro="year-date"/>
+          </group>
+        </if>
+        <!-- Generic/fallback format (citation) -->
+        <else>
+          <group delimiter=", ">
+            <group delimiter=", ">
+              <text macro="author-short"/>
+              <text macro="year-date"/>
+            </group>
+            <group>
+              <text term="page" form="short" suffix=" "/>
```

Comment:
Use label variable= locator here, so out works for other locator like chapter (sorry for formatting, typing on phone)